ansible/templates/centos7/omero-web-systemd.service.j2

```diff
@@ -14,9 +14,9 @@ Type=forking
 PIDFile={{ omero_user_home_dir }}/OMERO.py/var/django.pid
 Restart=no
 RestartSec=10
-Environment="PATH={{ virtualenv_path }}/bin:/bin:/usr/bin" "WEBBINDIR={{ omero_user_home_dir }}/OMERO.py/bin"
-ExecStart={{ virtualenv_path }}/python $WEBBINDIR/omero web start
-ExecStop={{ virtualenv_path }}/python $WEBBINDIR/omero web stop
+Environment="PATH={{ virtualenv_path }}/bin:/bin:/usr/bin"
+ExecStart={{ virtualenv_path }}/bin/python /home/omero/OMERO.py/bin/omero web start
```

Comment:
I think `/home/omero` should be `{{ omero_user_home_dir }}`

Reply:
forgot to re-establish that one
thanks for spotting it